hw/top_earlgrey/dv/chip_sim.core

```diff
@@ -7,32 +7,21 @@ description: "Earlgrey chip DV sim target"
 filesets:
   files_rtl:
     depend:
-      - lowrisc:systems:top_earlgrey:0.1
-      - lowrisc:ibex:ibex_tracer:0.1
-    files:
-      - ../rtl/top_earlgrey_asic.sv
-      - ../ip/xbar_main/dv/autogen/xbar_main_bind.sv
-      - ../ip/xbar_peri/dv/autogen/xbar_peri_bind.sv
```

Comment:
This is basically the motivation for this whole change - fusesoc does not support referencing sources in a different hierarchy.